dotnet-audit-logs-example/Views/Home/SendEvents.cshtml

```diff
@@ -3,105 +3,209 @@
-                    class='button nav-item blog-nav-button'>Blog</button></a>
-            <a href="https://workos.com/" target="_blank"><button class='button button-outline'>WorkOS</button></a>
-        </div>
-    </div>
-    <div class='flex'>
-        <div class="logged_in_div_left">
-            <div class="title-text">
-                <h1>Your app,</h1>
-                <h2 class="home-hero-gradient">Enterprise Ready</h2>
-            </div>
-            <div class="title-subtext">
-                <p>Start selling to enterprise customers with just a few lines of code.</p>
-                <p>Implement features like single sign-on in minutes instead of months.</p>
-            </div>
-            <div class="flex success-buttons">
-                <a href="https://workos.com/signup" target="_blank"><button class='button'>Get Started</button></a>
-                <a href="mailto:sales@workos.com?subject=WorkOS Sales Inquiry" target="_blank"><button
-                        class='button button-outline sales-button'>Contact
-                        Sales</button></a>
+            <hr style="height:0.5px;width:75%;">
+            <div class="flex flex_column success-buttons">
+                <a href="/logout">
+                    <div class="flex sidebar-button">
+                        <div><i icon-name="settings-2" class="stroke_width=1"></i></div>
+                        <div>
+                            <p>Organizations</p>
+                        </div>
+                    </div>
+                </a>
+                <div class="flex sidebar-button selected">
+                    <div><i icon-name="edit" class="stroke_width=1"></i></div>
+                    <div>
+                        <p>Audit Logs</p>
+                    </div>
+                </div>
             </div>
         </div>
         <div class="logged_in_div_right">
-            <div class="flex flex-start width-40vw">
-                <div><h1>@ViewData["OrgName"]</h1></div>
+            <div class="logged_in_nav">
+                <div class="flex">
+                    <div>
+                        <a href="https://workos.com/docs" target="_blank"><button class='button nav-item'>Documentation</button></a>
+                    </div>
+                    <div>
+                        <a href="https://workos.com/docs/reference" target="_blank"><button class='button nav-item'>API
+                            Reference</button></a>
+                    </div>
+                    <div>
+                        <a href="https://workos.com/blog" target="_blank"><button
+                            class='button nav-item blog-nav-button'>Blog</button></a>
+                    </div>
+                    <div>
+                        <a href="https://workos.com/" target="_blank">
+                            <img class="nav-image" src="../../wwwroot/images/workos_new.png" alt="link to workos.com">
```

Comment:
You don't need to include the full path with dotnet, you can just reference the images in the images folder like this: src="~/images/workos_logo_new.png"